ruff_lsp/server.py

```diff
@@ -1169,6 +1172,15 @@ async def _run_check_on_document(
         else:
             argv.append(arg)
 
+    # If the Ruff version is sufficiently recent, use the deprecated `--format`
+    # argument instead of `--output-format`.
```

Comment:
?
```suggestion
    # If the Ruff version is not sufficiently recent, use the deprecated `--format`
    # argument instead of `--output-format`.
```